src/components/header/header-link/header-link.ts

```diff
@@ -0,0 +1,72 @@
+import { spread } from '@open-wc/lit-helpers';
+import { nothing, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { html, unsafeStatic } from 'lit/static-html.js';
+
+import type { LinkRenderVariables } from '../../core/common-behaviors';
+import {
+  LanguageController,
+  SbbIconNameMixin,
+  resolveLinkOrStaticRenderVariables,
+  SbbLinkBaseElement,
+  targetsNewWindow,
+} from '../../core/common-behaviors';
+import { setAttributes } from '../../core/dom';
+import { i18nTargetOpensInNewWindow } from '../../core/i18n';
+import '../../icon';
+import { SbbHeaderActionCommonElementMixin } from '../common/header-action-common';
+
+/**
+ * It displays a link element that can be used in the `sbb-header` component.
+ *
+ * @slot icon - Slot used to render the link icon.
+ * @slot - Use the unnamed slot to add content to the `sbb-header-link`.
+ */
+@customElement('sbb-header-link')
+export class SbbHeaderLinkElement extends SbbHeaderActionCommonElementMixin(
+  SbbIconNameMixin(SbbLinkBaseElement),
+) {
+  private _language = new LanguageController(this);
+
+  protected override render(): TemplateResult {
+    const {
+      tagName: TAG_NAME,
+      attributes,
+      hostAttributes,
+    }: LinkRenderVariables = resolveLinkOrStaticRenderVariables(this);
+    setAttributes(this, hostAttributes);
+
+    /* eslint-disable lit/binding-positions */
+    return html`
+      <${unsafeStatic(TAG_NAME)} class="sbb-header-action" ${spread(attributes)}>
```

Comment:
I think we can remove the static variant here.

Reply:
Done in [9d4d702](https://github.com/lyne-design-system/lyne-components/pull/2364/commits/9d4d702b14963775aa5b8c002d1a9cd153f70c3f)